### PR TITLE
document scopes per-connector for MSFT sources 

### PR DESCRIPTION
**Pull request by eschultink**
### Features
 - add per-connector docs of required scopes.

This was rather tedious, I'm worried difficult to maintain; will see how Data Inventory project turns out, and probably refer customers to that (built programmatically) - than try to have this directly in proxy docs.

### Change implications

 - dependencies added/changed? **no**
 - something important to note in future release notes? **no**

### CODE REVIEW

File: docs/sources/microsoft-365/msft-teams/README.md [hunk 1]
```diff
@@ -1,14 +1,42 @@
-# MSFT Teams
+# Micorosft Teams
```
Comment (aperez-worklytics) [inline docs/sources/microsoft-365/msft-teams/README.md:1]:
```suggestion
# Microsoft Teams
```

File: docs/sources/microsoft-365/msft-teams/README.md [hunk 2]
```diff
@@ -1,14 +1,42 @@
...
 
-## Examples
+Connect Micorosft Teams data to Worklytics, enabling communication analysis and general collaboration
```
Comment (aperez-worklytics) [inline docs/sources/microsoft-365/msft-teams/README.md:3]:
```suggestion
Connect Microsoft Teams data to Worklytics, enabling communication analysis and general collaboration
```

File: docs/sources/microsoft-365/msft-teams/README.md [hunk 3]
```diff
@@ -1,14 +1,42 @@
...
+insights based on collaboration via Micorosft Teams. Includes user enumeration to support fetching
```
Comment (aperez-worklytics) [inline docs/sources/microsoft-365/msft-teams/README.md:4]:
```suggestion
insights based on collaboration via Microsoft Teams. Includes user enumeration to support fetching
```

**Review by aperez-worklytics — APPROVED** [review]

**Review by davidfq — APPROVED** [review]

File: docs/sources/microsoft-365/entra-id/README.md [hunk 4]
```diff
@@ -1,15 +1,39 @@
...
+`UserPrincipalName` (often your email address).
 
 See more examples in the `docs/sources/microsoft-365/entra-id/example-api-responses` folder
 of the [Psoxy repository](https://github.com/Worklytics/psoxy).
+
+## Sanitization Rule Examples
+
+- [Default Rules](entra-id.yaml)
+- [Rules, pseudonymizing MSFT account IDs](entra-id_no-app-ids.yaml)
+- [Rules, pseudonimizing MSFT account IDs](entra-id_no-app-ids_no-orig.yaml)
```
Comment (jlorper) [inline docs/sources/microsoft-365/entra-id/README.md:38]:
```suggestion
- [Rules, pseudonymizing MSFT account IDs](entra-id_no-app-ids.yaml)
- [Rules, pseudonymizing MSFT account IDs](entra-id_no-app-ids_no-orig.yaml)
```
Also, the title is same in both... explain the difference there? the no orig thing?

**Review by jlorper — APPROVED** [review]
Typos, other than that OK

**Pull request merged**